step5/tictactoe.js

```diff
@@ -0,0 +1,60 @@
+CollectionBoxes = new Meteor.Collection('boxes');
+
+if (Meteor.isClient) {
+  Session.set({currentPlayer: 'X'});
+
+  var currentPlayer = function(){
+    return Session.get('currentPlayer');
+  };
+
+  var setNextPlayer = function(){
+    if(currentPlayer() == 'X') Session.set({currentPlayer: 'O'});
+    else Session.set({currentPlayer: 'X'});
+  };
+
+  Meteor.startup(function(){
+    console.log('started up client');
+  });
+
+  Template.gameboard.events({
+    click: function() {
+      console.log('Clicked the gameboard');
+    }
+  });
+
+  Template.gameboard.helpers({
+    boxes: function(){
+      return CollectionBoxes.find({});
+    },
+    currentPlayer: currentPlayer
+  });
+
+  Template.box.events({
+    click: function() {
+      var cellFilled = this.player;
+      if (cellFilled) {
+        return;
+      }
+      var player = currentPlayer();
+      CollectionBoxes.update(this._id, { $set: { player: player } });
```

Comment:
Dit is de code voor stap6. Hier kunnen we alleen een Session variabele opvragen on click. Of zelfs zetten on click:
```javascript
Template.box.helpers({
      player: function(){
        return Session.get('player');
      }
    });

    Template.box.events({
      'click': function(){
        return Session.set('player', 'X');
      }
    });
```